tokio-stream/src/lib.rs

```diff
@@ -81,6 +81,15 @@ pub mod wrappers;
 
 mod stream_ext;
 pub use stream_ext::{collect::FromStream, StreamExt};
+/// Adapters for types that implement the `Stream` trait.
+/// Created through the tokio [`StreamExt`] methods.
+pub mod adapters {
+    pub use crate::stream_ext::{
+        AllFuture, AnyFuture, Chain, Filter, FilterMap, FoldFuture, Fuse, Map, MapWhile, Merge,
+        Next, Peekable, Skip, SkipWhile, Take, TakeWhile, Then, TryNext,
+    };
```

Comment:
You are missing: `Collect`, `TimeoutRepeating`, `Throttle`, `ChunksTimeout`.

Reply:
`Collect` states in its documentation
> Do not export this struct until `FromStream` can be unsealed.
I'm doing the others ones right now.

Reply:
Ok, that's fine.